### DISCUSSION TITLE
Empty screen without any issue when was added custom domain to gitHub pages

### DISCUSSION THREAD

**Discussion by HowFastGot** [community/community#38990] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, everyone !
My name is Eugene and I need some assitance with troubleshooting of custom domain.

Three day ago I bought a domain name, configure DNS setting on CloudFlare DNS provider (screen URL - https://fex.net/ru/s/cckfazc ) accordin to gitHub's docs and add custom domain to my gitHub pages. 
On DNS provider plarform tech support assure me that everthig configure correctly, domain is active.

But when I follow this domain, I see only white empty screen without any errors.  How I can fix it ?
Apprecite your assistance !

Best Regards,
Evgeny

**Comment by jfullstackdev** [comment]
yup, it's already running, if this is the correct site you are referring too

![image](https://user-images.githubusercontent.com/47092464/201522742-05501061-08e1-433b-a740-5110bfe6f8ba.png)

but as you can see at the Developer Tools, others are 404, I'm not very sure of that, do you fetch JavaScript in there ?

**Reply by HowFastGot** [reply]
> yup, it's already running, if this is the correct site you are referring too
> 
> ![image](https://user-images.githubusercontent.com/47092464/201522742-05501061-08e1-433b-a740-5110bfe6f8ba.png)
> 
> but as you can see at the Developer Tools, others are 404, I'm not very sure of that, do you fetch JavaScript in there ?

Thank you for response !

Correnct domain name  - https://vovk-digital.com

Yes, I have discover that I have turn off the errors message in conslole. Now I see these errors - the server responded with a status of 404 ().
Will do my best to  fix it.  May you know some way for it ?

**Comment by dhrubasaha08** [comment, marked as answer]
@HowFastGot Hey Eugene make your <username>.github.io repository visibility as Public. Then go  to <username>.github.io !

**Reply by HowFastGot** [reply]
Thank you for this assistance !

My repository was with public visibility but I have understood that when I published it for the first time in a JSONS file in the field "homepage" I marked the default path with " .github.io ", now I have changed this field to custom domain and now it works.

Everybody, thank you for your replies !

**Reply by dhrubasaha08** [reply]
You're welcome!
